src/docs/build-windows.dox

```diff
@@ -101,4 +101,8 @@ compression; see @ref compression for more information.
 Configure WiredTiger for <a href="http://www.zlib.net/">zlib</a>
 compression; see @ref compression for more information.
 
+@par \c --disable-standalone-build
```

Comment:
Can you add a similar doc change for Linux variant also?

Reply:
Updated `build-posix.dox`.